com.endlessm.Sdk.json.in

```diff
@@ -19,8 +19,12 @@
                           "touch -d @0 /usr/share/fonts/*",
                           "fc-cache -fs"
                         ],
-    "cleanup-platform": [ "/include",
+    "cleanup-platform": [ "/bin/basin",
```

Comment:
You should move `cleanup-platform` rules for Basin directly in the `basin` module; the `cleanup-platform` rules at the top-level are meant for top-level items.